spec/factories/factories.rb

```diff
@@ -0,0 +1,8 @@
+include ActionDispatch::TestProcess
+FactoryGirl.define do
+  factory :test_photo, class: Photo do
+    image { fixture_file_upload(
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.